spec/models/king_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe King, type: :model do
+  
+  describe "King's valid_move" do
+  	it "should be inside destination, blocked" do
+      game = create :game
```

Comment:
Since you're doing this as part of every test, you should make the code a bit more DRY by using `let` to create the `game` variable.